scripts/buildocaml.js

```diff
@@ -22,13 +22,18 @@ function getVersionPrefix() {
     cached = version.substr(0, version.indexOf("+"));
     return cached;
   }
+  console.error(`cannot find '${file}'`);
 
   file = path.join(__dirname, "..", "OCAML_VERSION");
   if (fs.existsSync(file)) {
     var version = fs.readFileSync(file, "ascii");
     cached = version.substr(0, version.indexOf("+"));
     return cached;
   }
+  console.error(`cannot find '${file}'`);
+
+  console.error("You should create OCAML_VERSION or ocaml/VERSION file to specify OCaml version like '4.02.3+buckle-master'");
+  console.error("For example, run `opam switch show > OCAML_VERSION`");
   throw new Error("version file not found");
```

Comment:
let's not mention opam here, opam is not a dependency, can we just give a sample as
```
bucklescript>cat ocaml/VERSION 
4.02.3+BS

# The version string is the first line of this file.
# It must be in the format described in stdlib/sys.mli
```